content-src/components/ActivityFeed/ActivityFeed.scss

```diff
@@ -64,6 +68,58 @@
     margin-top: -($feed-gutter-v / 2 + 18 / 2);
     cursor: pointer;
   }
+
+  .action-items-container {
+    background: $white;
+    border: solid 1px $light-grey;
+    border-radius: $border-radius;
+    position: absolute;
+    right: -5px;
+    bottom: -5px;
+    display: flex;
+    flex-direction: row;
+    align-items: flex-end;
+    justify-content: center;
+    opacity: 0;
+  }
+
+  .action-item {
+    cursor: pointer;
+    width: 40px;
+    height: 32px;
+    background-position: center center;
+    background-repeat: no-repeat;
+    border-right: solid 1px $light-grey;
```

Comment:
Actually scratch that, use `$faintest-black` so that the line on the side can fade through